main.py

```diff
@@ -316,6 +317,13 @@ def ui_access_wrapper(self, *args, **kwargs):
         editor.StartedTutorialEventHandler, 'started_tutorial_event_handler'),
 
     get_redirect_route(
+        r'/collection_editor/data/<collection_id>', collection_viewer.CollectionDataHandler,
```

Comment:
Is this redirect route needed anymore? It seems to be the same as the one for the learner view, except it isn't used.